packages/schematics/src/exception/exception.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export class BaseException extends Error {
```

Comment:
I assume you're building down to ES2015 and requiring Node 7+ (I think), then? Because the only problems come if you're targeting ES5. If that's the case, you just need to add `Object.setPrototypeof(this, BaseException.prototype)` in the `constructor`, and you'll be fine in any version of Node.